webpack.config.js

```diff
@@ -180,6 +180,7 @@ module.exports = {
         {
           from: 'src/documentation/api',
           to: 'documentation/api',
+          noErrorOnMissing: true,
```

Comment:
@mademoiselle-chantal , why did you this?

Reply:
I was getting the error below
<img width="1470" alt="Screenshot 2024-03-31 at 21 31 34" src="https://github.com/egor-progger/pointlinejs/assets/120084609/c2d4ef8a-5c00-48e5-b9a9-152200d91986">

so that line of code solved it.

Reply:
Did you start dev server with command `npx webpack serve`?
I updated [How to run dev-server for debugging](https://github.com/egor-progger/pointlinejs#how-to-run-dev-server-for-debugging) section. Now start dev server run command:
```
npm run start
```
Delete `noErrorOnMissing` and check `npm run start`. If no errors, commit webpack.config.js without `noErrorOnMissing` please.

Reply:
I made the changes.

Reply:
I made the changes.